canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -245,7 +245,7 @@ class VirtualizedTable extends React.Component {
 			? null
 			: (<div className="properties-vt-info-icon-tip">
 				<Tooltip
-					id={`properties-tooltip-${columnData.headerLabel}-info`}
+					id={`properties-tooltip-${columnData.headerLabel.split(" ").join("")}-info`}
```

Comment:
This is the same as `string.replace(" ", "")`?

Reply:
This doesn't seem like a good way of creating ids.  There could be other chars in the header label that cause issues and there is no guarantee these are unique in the DOM.  Can we generate a guid for each header instead?  We wouldn't want to generate it on each rerender though.